### ISSUE TITLE
Plan to release your pre-trained models

### ISSUE THREAD

**Issue by MINGUKKANG** [PeterouZh/Omni-GAN-PyTorch#3]
Hi,

Thank you for your great work.

I read your paper twice (before revision and after revision). I think that your paper explains differences between projection-based models and classifier-based models. To enjoy strengths of the projection and classifier-based models, you have proposed a novel OmniGAN and an extra weight decay regularization. The results from your work are very impressive!

I want to know your plan to release pre-trained models and details of training. 

Also, I think it is good to visualize qualitative results sampled from OmniGAN on ImageNet 128. 

Thank you.

Best,

Minguk

**Comment by PeterouZh** [comment]
Hi, 

Thank you for your interest! 

There is an updated version of my paper, and I will put it on arXiv as soon as this week. We introduce an enhanced Omni-GAN, named Omni-INR-GAN. The training details are placed in the appendix of the updated version (v3), which you can refer to. When my paper is accepted, I will release all the code and pre-trained models. 

Thank you again.

Best,
Peterou
